save-backend/src/main/kotlin/com/saveourtool/save/backend/service/UserDetailsService.kt

```diff
@@ -23,46 +25,48 @@ class UserDetailsService(
 ) : ReactiveUserDetailsService {
     private val logger = LoggerFactory.getLogger(javaClass)
 
-    override fun findByUsername(username: String): Mono<UserDetails> = Mono.fromCallable {
-        userRepository.findByName(username)
-    }.getIdentitySourceAwareUserDetails(username)
+    override fun findByUsername(username: String): Mono<UserDetails> =
+            { userRepository.findByName(username) }
```

Comment:
Why is it wrapped into brackets?

Reply:
it's extension for `Mono.fromCallable` for Kotlin -- this return `Mono<T>` instead of `Mono<T?>`